crypto/fipsmodule/cpucap/cpu_aarch64.c

```diff
@@ -26,7 +26,7 @@ void handle_cpu_env(uint32_t *out, const char *in) {
 
   // Detect if the user is trying to use the environment variable to set
   // a capability that is _not_ available on the CPU:
-  // If getauxval() returned a non-zero hwcap in `armcap` (out)
+  // If the runtime capability check (e.g via getauxval() on Linux) returned a non-zero hwcap in `armcap` (out)
```

Comment:
Line length

Reply:
Yikes, I misunderstood the previous commit suggestion for this line! Will add 👍